chart.py

```diff
@@ -40,7 +40,9 @@ def glucose_line_chart(recent_glucose, predicted_glucose, targets):
                 targets
             ))
 
-        for entry in predicted_glucose:
+        iter_predicted_glucose = iter(predicted_glucose)
+        next(iter_predicted_glucose)    
```

Comment:
Yeah, see that now. `for entry in predicted_glucose[1:]` won't have that
issue, correct?

On Thu, Sep 3, 2015 at 3:55 PM, Nathan Racklyeft <notifications@github.com>
wrote:

> In chart.py
> <https://github.com/loudnate/openaps-monitor/pull/15#discussion_r38706159>
> :
>
> > @@ -40,7 +40,9 @@ def glucose_line_chart(recent_glucose, predicted_glucose, targets):
> >                  targets
> >              ))
> >
> > -        for entry in predicted_glucose:
> > +        iter_predicted_glucose = iter(predicted_glucose)
> > +        next(iter_predicted_glucose)
>
> This will raise an exception if predicted_glucose is empty
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loudnate/openaps-monitor/pull/15/files#r38706159>.
>



-- 
Chris Hannemann
chris.hannemann@gmail.com